db/mysql/schema.sql

```diff
@@ -438,6 +438,8 @@ ADD FOREIGN KEY (`user_id`)
     REFERENCES `usr_user` (`id`)
 	ON UPDATE CASCADE ON DELETE CASCADE;
 
+ALTER TABLE ppl_model ADD UNIQUE (name(255));
```

Comment:
The limit should be the same as the limit in cz.cuni.mff.xrg.odcs.commons.app.constants.LengthLimits.PIPELINE_NAME, which is enforced in GUI